subsys/net/hostname.c

```diff
@@ -17,14 +17,14 @@ LOG_MODULE_REGISTER(net_hostname, CONFIG_NET_HOSTNAME_LOG_LEVEL);
 #include <zephyr/net/net_core.h>
 #include <zephyr/net/net_mgmt.h>
 
-static char hostname[NET_HOSTNAME_SIZE];
+static char _hostname[NET_HOSTNAME_SIZE];
```

Comment:
It was requested not to use underscore prefix for variable names.

Reply:
What is the preferred naming style when I want to keep the `hostname` argument in `net_hostname_set`? Is `g_hostname` ok?

Reply:
For example `char *host` would be fine as a function argument.